app/controllers/pizzas_controller.rb

```diff
@@ -0,0 +1,31 @@
+class PizzasController < ApplicationController
+  def new
+    puts params
```

Comment:
3 weeks of asking you to stop committing this!!!!! If I see this on the actual code challenge, I'm going to fail you immediately.